index.js

```diff
@@ -1,30 +1,54 @@
-import Audience from './models/audience'
-import { BigQuery } from '@google-cloud/bigquery'
-import { handler } from './handlers/tealium'
-import rollbar from './config/rollbar'
+import Audience from "./models/audience";
+import { BigQuery } from "@google-cloud/bigquery";
+import { handler, bqHandler } from "./handlers/tealium";
+import rollbar from "./config/rollbar";
```

Comment:
Looks like your IDE changed quotes. This project uses `standardjs`, last I checked it still preferred single quotes.